Week1/js1-w1-A.A.A.js

```diff
@@ -0,0 +1,175 @@
+// Greetings
+
+console.log ('Hello World!'); //English
+
+console.log ('Halo, Dunia!'); // Indonesian
+
+console.log ('Ciao Mondo!'); // Italian
+
+console.log ('Hola Mundo!'); // Spanish
+
+
+// Debug single quote issue
+
+console.log('I\'m awesome');
+
+// Variables & Values
+
+var x;
+
+console.log("the value of my variable x will be: a number");
```

Comment:
I don't think the value of `x` will be `a number`, are you sure this is correct?